packages/ffe-cards/less/ffe-new-card.less

```diff
@@ -0,0 +1,107 @@
+.ffe-new-card {
+    background: @ffe-white;
+    box-shadow: 0 1px 4px 0 rgba(38, 38, 38, .3);
+    display: flex;
+    flex-flow: row nowrap;
+    margin: 0 0 10px;
+    padding: 10px 20px;
+    border: 2px solid @ffe-white;
+    border-radius: 5px;
+    outline: none;
+    cursor: pointer;
```

Comment:
Should all cards be clickable?
If not, I suppose we only want the cursor on clickable cards.
This is potentially something the consumer could be responsible for.

Reply:
Most cards should probably be clickable. Therefore I think using a `<button />` as the default Element in the React implementation might be a good idea. This way, we get a lot for free! Remember to reset `appearance` if you do though.

Reply:
You already accept `element` as a prop, if it's supposed to be a link, perhaps the consumer can provide `"a"` as the value and the cursor should act accordingly (I'd assume).

Reply:
As per design definition, ALL cards should be clickable. Anything else is just a box grouping a set of information.

Reply:
Thanks for clearing that up, Adi.

I see I need to add `text-align: left` if button is used as element.

Also, if element is `"a"` I get the pointer on the entire box by default, but as they should all be clickable, I think we should keep the cursor styling.